portfolio/src/main/webapp/index.html

```diff
@@ -17,10 +18,28 @@
           <li> <a href="#">Projects</a></li>
           <li> <a href="#">About</a></li>
           <li> <a href="#">Contact</a></li>
-          <li> <a href="#">Resume</a></li>
+          <li> <a href="Angel_Walia_Resume.pdf">Resume</a></li>
         </ul>
       </nav>
-    </section>    
+
+      <div class="text-container">
+       <p>Hello there,</p>
+       <p>I am Angel!</p>
+       <p>I am a Software Engineer and Designer</p> 
+       
+       <button class="resume-button" onclick="window.open('Angel_Walia_Resume.pdf')">Download My Resume</button>
+      
```

Comment:
the space on line 29, 31 is intentional?

Reply:
I guess I am just used to write code in a very spaced way :( I will be mindful hence

Reply:
I think that's alright. As long as you are consistent with the way the code is written and the readability is intact, it is all good :)